tests/mpi_pi.f90

```diff
@@ -30,7 +30,7 @@ program calculatePi
 
   if (worldRank == 0) then
     call date_and_time(VALUES=timeValues)
-    write(*,"(A,I2,A,I2,A,I2.2)") "Start Time = ", timeValues(5), ":", timeValues(6), ":", timeValues(7)
+    write(*,"(A,I2.2,A,I2.2,A,I4,A,I2,A,I2,A,I2.2)") "Start Time = ", timeValues(3), "/", timeValues(2), "/", timeValues(3), " ", timeValues(5), ":", timeValues(6), ":", timeValues(7)
```

Comment:
Still trying to figure out where the first 3 numbers of the year aren't showing up, but the format looks much better now.

Reply:
For the year, I think you need `I4` for a 4-digit year.  The `I2.2` is for two-digit numbers that may start with 0.

Reply:
I believe that it is in the correct place, `(A,I2.2,A,I2.2,A,I4,A,I2,A,I2,A,I2.2` since the first is `I2.2` for the day -- I left this with the `.2` for the first 9 days of the month format, the second is `I2.2` is for the month and the third is `I4` for the year. 

I'll research more into Fortran and how to write these values and what order they should appear in.

Reply:
You seem to have `timeValues(3), "/", timeValues(2), "/", timeValues(3)` in the code.  The last one needs to be something other than 3.  It's printing the day again with leading spaces... `   4`

Reply:
I think my initial code was bad.  You'll want `2.2` for all of the 2-digit fields because any of them could start with zero.  So, the hour and minute need them too.

Reply:
I see what you mean now, I was looking at the wrong thing initially. That bug would have been much easier to see if the day had not been the 4th. I updated that and see it's printing correctly now as `Start Time = 05/01/2024 19:27:26`. I'll push that shortly and incorporate your other comment as well.